samples/matter/contact_sensor/sysbuild/mcuboot/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
+
+# The following configurations are required to support simultaneous multi image update
+CONFIG_PCD_APP=y
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
+
+CONFIG_BOOT_SWAP_USING_MOVE=n
+# Multi-image updates do not support image swapping yet.
+CONFIG_BOOT_UPGRADE_ONLY=y
+
+# The network core cannot access external flash directly. The flash simulator must be used to
+# provide a memory region that is used to forward the new firmware to the network core.
+CONFIG_FLASH_SIMULATOR=y
+CONFIG_FLASH_SIMULATOR_DOUBLE_WRITES=y
+CONFIG_FLASH_SIMULATOR_STATS=n
+
+CONFIG_ZCBOR=y
```

Comment:
why is cbor enabled in MCUboot?

Reply:
I have no idea, originally this config appeared in our configuration for nRF53 only with this commit: https://github.com/nrfconnect/sdk-connectedhomeip/commit/d3d1862f885b5ddbbcc96f4ccaf57b608bb55e41

So it seems at that point it was required to compile mcuboot with nRF53. Now it's working after deleting this option, so I will remove it.